.github/workflows/ci.yml

```diff
@@ -82,6 +77,63 @@ jobs:
       - name: Verify working directory is clean
         run: git diff --exit-code
 
+  # States that we want to ensure can be built, but that we don't actively run tests for.
+  check-msrv:
+    name: >
+      Check${{
+      matrix.state != 'NOT_A_PUZZLE' && format(' {0}', matrix.state) || ''
+      }} build on ${{ matrix.target }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.state != 'NOT_A_PUZZLE' }}
+    strategy:
+      matrix:
+        target:
+          - Linux
+          - macOS
+          - Windows
+        state:
+          - ZFuture
+
+        include:
+          - target: Linux
+            os: ubuntu-latest
+          - target: macOS
+            os: macOS-latest
+          - target: Windows
+            os: windows-latest
+
+          - state: ZFuture
+            extra_flags: zfuture
```

Comment:
Weird formatting?

Reply:
Intentional: "ZFuture" is the state name (shown in CI status name), `zfuture` is (what was) the feature flag.